addons/im_livechat/static/src/js/im_livechat.js

```diff
@@ -449,22 +449,21 @@ var Feedback = Widget.extend({
      * @private
      * @param {Object} options
      */
-    _sendFeedback: function (options) {
+    _sendFeedback: function (reason) {
         var self = this;
         var args = {
             uuid: this._livechat.getUUID(),
             rate: this.rating,
-            reason : options.reason
+            reason : reason
```

Comment:
simply `reason,` is enough:

```js
var args = {
   // ...
  reason,
}
```

Reply:
??

Reply:
ha yes, `reason` and not `reason: reason`

Reply:
yup :)